content/05.discussion.md

```diff
@@ -12,22 +12,22 @@ We then compared this to the authorship statistics from _Nature_'s papers and a
+This finding, coupled with the near equal number of articles written by male and female predicted journalists, argues for more diversity in topical coverage. 
+"Career Feature" articles highlight current topics relevant to working scientists and frequently highlight systemic issues with the scientific environment.
 This column allows space for marginalized people to critique the current state of affairs in science or share their personal stories.
 This type of content encourages the journalist to seek out a diverse set of primary sources.
-Including more content that is not primarily focused on recent publications but all topics surrounding the practice of science can serve as one tool to rapidly achieve gender parity in journalistic recognition.
+Including more content that is not primarily focused on recent publications, but all topics surrounding the practice of science, can serve as an additional tool to rapidly achieve gender parity in journalistic recognition.
 
 
-In addition to comparing gender disparity to the general population, we also compared against the relative proportion of authors and speakers predicted to be men.
-We found a slight over-representation of men and that the disparity is dependent on the authorship position and the year.
-Before 2010, quotes predicted to be from men were overrepresented in comparison to both first and last authors, but between 2010 and 2017, quotes predicted to be from men were only overrepresented in comparison to first authors.
-In 2020, we find a slight over-representation of quotes from predicted women relative to first and last authors, but still severely under-represented when considering the general population.
+When considering the relative proportion of authors and speakers predicted to be men, we only find a slight over-representation of men. This overrepresentation is dependent on the authorship position and the year.
+Before 2010, predicted male quotes are overrepresented in comparison to both first and last authors, but between 2010 and 2017 predicted male quotes are only overrepresented in comparison for first authors.
+In 2020, we find a slight over-representation of female quotes relative to first and last authors, but still severely under-represented when considering the general population.
 The choice of comparison between first and last authors can reveal different aspects of the current state of academia.
 While this does not hold in all scientific fields, first authors are typically early career scientists and last authors are more senior scientists.
+It has also been shown that early career scientists tend to be more diverse than senior scientists [@doi:10.7554/eLife.60829; @doi:10.1096/fj.201800639].
+Since we find that quotes are only slightly more likely to come from a last author, it is reasonable to compare the relative rate of predicted male quotes to either authorship position.
```

Comment:
```suggestion
Since we find that quotes are only slightly more likely to come from a last author, it is reasonable to compare the relative rate of predicted quotes from men to either authorship position.
```